### ISSUE TITLE
Reader scrolls to anchors but does not set focus on target element.

### ISSUE THREAD

**Issue by wilsmic2** [ManifoldScholar/manifold#2926]
When using footnotes in a book, if you click on a footnote, which will redirect you to the "Notes section" of the book, the focus does not land at the correct place in the newly loaded page. For example, if I click on the 5th footnote in Chapter 1, I should be taken to the link of the 5th footnote from Chapter 1 in the "Notes" page, and although the visible window scrolls to the correct place in the page, the focus remains at the top of the page, meaning that screen reader users are not able to immediately read the footnote that they want to and would have to tab through the entire page to reach it.

## Current Behavior
Clicking "Enter" on a footnote in a book, which loads the "Notes" page in the current tab, will result in the focus landing at the incorrect spot on the newly loaded page. Also note that this happens when going from the "Notes" page back to the reference in the Chapter itself. If you open the "Notes" page in a separate tab or window using CTRL + SHIFT + enter or SHIFT + enter, then the focus will land at the expected location, being directly at the footnote that is being referenced rather than at the top of the list.

## Expected Behavior
The expected behavior is that the focus when clicking on footnotes that navigate to and from the "Notes" section of a book will be on the footnote item that is being referenced rather than at the top of the page.

## Steps to Reproduce
1. Navigate to any book that has footnotes - an example can be found here: https://uw.manifoldapp.org/read/the-power-of-the-brush-ec9fe559-4cfc-4e01-beb3-7178806966ef/section/a34f4e5a-9585-4ac3-a0ce-68305b974351
2. Tab to a footnote in the chapter.
3. Use "Enter" to follow the footnote link and see where your focus lands in the "Notes" page.
4. Navigate back to to the footnote reference in the Chapter and use "CTRL + SHIFT + enter" or "SHIFT + Enter" to follow the footnote link and notice how the focus now lands at the correct place.
5. Attempt to navigate back to the main book using the links in the "Notes" page and notice how the same behavior is observed.

If you have any questions feel free to ask.

**Comment by mkgold** [comment]
Thank you for reporting this @wilsmic2 !

**Comment by zdavis** [comment]
I'm renaming this issue to accurately describe the problem. The issue is not about footnotes, per se, but rather about how the reader handles anchor links in general.

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_

**Comment by tsangaaron09** [comment]
The UW ATX team has found additional use cases related to this issue found in student-created projects. In this use case, when we are using a screenreader to navigate anchor tags in the projects, selecting the anchor tag will visually bring you down to the footnotes at the end of the page, but once you try to navigate forward to read the footnote (NVDA and VO), the announcement will jump back to the main text. So, while the visuals bring the user to the footnotes, the screenreader retains its focus in the main text. We thought that it would be worthwhile to include this use case as the issue with anchor tags also persist if the footnotes are on the same page as the main text as opposed to on different pages as described in the original issue. 

## Steps to Reproduce ##
1. Navigate to this link for the student project: https://uw.manifoldapp.org/read/imaginingtheunseen/section/f627cce4-1dc4-4d47-8f36-0c1424485e09#fn1
2. Navigate to any link in the book that references the footnotes
3. Use "Enter" to follow the footnote link and see where your focus lands
4. Navigate to the next or previous element and notice how your screen reader will read the next or previous element to the anchor tag and not the footnote.

If you have any questions feel free to ask.

**Comment by zdavis** [comment]
That sounds like the same issue described above. We've modified the reader so that when it scrolls to an element, that element will receive focus if it's a focusable element. This fix will likely be moved into master today.

**Comment by zdavis** [comment]
Resolved in 8e93a8d4b237d5734af2c65398e3ff0d5fdac6ea
